bin/node/runtime/src/lib.rs

```diff
@@ -1247,7 +1248,7 @@ pub type SignedExtra = (
 	frame_system::CheckGenesis<Runtime>,
 	frame_system::CheckEra<Runtime>,
 	frame_system::CheckNonce<Runtime>,
-	frame_system::CheckWeight<Runtime>,
+	AdjustPriority<frame_system::CheckWeight<Runtime>, Divide, WeightPriorityAdjustment>,
```

Comment:
I think syntactically it would look much better if you make this be: 

```suggestion
	AdjustPriority<frame_system::CheckWeight<Runtime>, Divide, ConstU32<10>,
```

which reads: *divide* the priority of `CheckWeight` by `10`. The current `pub const WeightPriorityAdjustment: u64 = MAXIMUM_BLOCK_WEIGHT / 10;` is a bit ambiguous IMO.